docs/layers/language-server-protocol.md

```diff
@@ -97,6 +105,7 @@ default language server commands:
 | language     | server command                                   |
 | ------------ | ------------------------------------------------ |
 | `javascript` | `['javascript-typescript-stdio']`                |
+| `typescript` | `['typescript-language-server', '--stdio']`      |
```

Comment:
this is adding typescript support?

Reply:
yeah, ts lsp support has been added in old verison, but the doc is missing.